roles/swift-common/tasks/main.yml

```diff
@@ -76,6 +76,13 @@
     - logging
   when: logging.enabled|default('True')|bool
 
+- name: change swift log file permissions
+  file: path={{ item }} mode=0644 owner=swift group=swift state=touch
+  with_items:
+    - /var/log/swift/account.log
+    - /var/log/swift/container.log
+    - /var/log/swift/object.log  
+
```

Comment:
Actually you should just use variable for log paths defined in  https://github.com/blueboxgroup/ursula/blob/e3d19e68f8f0b42b05d0a4b34dc7eec7755ac8d4/roles/swift-common/defaults/main.yml